tests/api/test_aws.py

```diff
@@ -293,7 +293,7 @@ def get_statements_by_sid(policy_document):
 @pytest.mark.parametrize(
     'resources',
     [
-        ([],),
+        ([]),
```

Comment:
any reason to remove the comma? Isn't python gonna get confused and not understand this is a tuple?

Reply:
No, in fact it's probably bad practice to. In this case, I don't think it matters, but I'll revert the change.

Reply:
I should apply this in the following line too.

Reply:
FYI: I think without the comma Python *will* get confused and interpret that as a list rather than a tuple, e.g.:

```python
>>> ([],).__class__
<class 'tuple'> # correct

>>> ([]).__class__
<class 'list'> # probably not you want
```